tenant-cd/src/main/java/ai/vespa/hosted/cd/Endpoint.java

```diff
@@ -1,29 +1,33 @@
 package ai.vespa.hosted.cd;
 
+import ai.vespa.hosted.api.EndpointAuthenticator;
 import ai.vespa.hosted.cd.metric.Metrics;
 
 import java.net.URI;
+import java.net.http.HttpClient;
 import java.net.http.HttpRequest;
 import java.net.http.HttpResponse;
 
+import static java.nio.charset.StandardCharsets.UTF_8;
+
 /**
  * An endpoint in a Vespa application {@link Deployment}, which allows document and metrics retrieval.
  *
- * The endpoint translates {@link Query}s to {@link Search}s, and {@link Selection}s to {@link Visit}s.
- * It also supplies {@link Metrics}.
- *
  * @author jonmv
  */
 public interface Endpoint {
 
-    URI uri();
+    /** Returns the URI of the endpoint, with scheme, host and port. */
+    URI hostUri();
```

Comment:
Do you export more uris? Is that why it's called hostUri? I thought uri was nicer. Not important at this point, though

Reply:
Hmm, ok, maybe you're right :)

Reply:
With doc that's better, I agree.